specs/fault-proof/stage-one/optimism-portal.md

```diff
@@ -262,6 +375,10 @@ Allows a user to [prove](#proven-withdrawal) a withdrawal transaction.
   [Respected Game](./anchor-state-registry.md#respected-game).
 - MUST revert if the withdrawal is being proven against a game that has resolved in favor of the
   Challenger.
+- MUST revert if `superRootsActive` is `true`.
+- MUST revert if the proof provided by the user of the contents of the Output Root that the dispute
+  game argues about is invalid. This proof is verified by hashing the user-provided contents and
+  comparing them to the root claim of the referenced dispute game.
```

Comment:
As above, preimage of the Output Root is probably clearer than contents since the Output Root is a 32 byte hash.